public/data/ankanbag101.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "Ankan Bag",
+    "type": "personal",
+    "bio": "An alien👽 trying to make a difference in the tech world🔥",
+    "avatar": "https://github.com/ankanbag101.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/ankanbag101",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/AnkanBag06",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://linkedin.com/in/ankan-bag-74a238222/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Email me",
+        "url": "mailto:bagankan@gmail.com"
```

Comment:
Do you not want to choose an icon for this one? We have an envelope

Reply:
Thanks, I will add the envelope